src/store/flow/block.ts

```diff
@@ -71,12 +71,27 @@ export const mutations: MutationTree<IFlowsState> = {
     findBlockOnActiveFlowWith(blockId, state as unknown as IContext).config = {...currentConfig}
   },
   block_removeConfigByKey(state, {blockId, key}: { blockId: string, key: string}) {
-    Vue.delete(findBlockOnActiveFlowWith(blockId, state as unknown as IContext).config, key)
+    const base = findBlockOnActiveFlowWith(blockId, state as unknown as IContext).config
```

Comment:
Voto5 needs the ability to remove deep attributes from config, i.e. `vendor_metadata.io_viamo.*`

Reply:
Thanks for updating this @object-required , 💯